components/css/componentStyle/Badge.css

```diff
@@ -0,0 +1,138 @@
+.badge__items{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.badges{
+    margin-left: 4rem;
+    position: relative;
+}
+
+.action_btn{
+    width: 12rem;
+    padding: 1rem;
+    border: none;
+    background-color: #3f51b5;
+    color: white;
+    font-weight: 600;
+    border-radius: 4px;
+}
+
+.action_btn.home{
+    background-color:teal;
+}
+
+.badge1{
+    height: 2rem;
+    width: 2rem;
+    border-radius: 50%;
+    background-color: tomato;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem;
+    color: white;
+    position: absolute;
+    top: -1rem;
+    left: -1rem;
```

Comment:
avoid repetition of code, if you require the same code in multiple classes put that code in some util class and use it wherever you need it.

Reply:
i will improve this